test/js-native-api/test_instance_data/test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../../common');
+
+// Testing api calls for instance data
+const test_instance_data =
+  require(`./build/${common.buildType}/test_instance_data`);
+
+if (process.argv[2] === 'child') {
+  test_instance_data.setPrintOnDelete();
+  return;
+}
+
+const { spawnSync } = require('child_process');
+const assert = require('assert');
+
+// Test that instance data can be accessed from a binding.
+assert.strictEqual(test_instance_data.increment(), 42);
+
+// Test that the instance data finalizer gets called.
+assert.strictEqual(
+  (spawnSync(process.execPath, [__filename, 'child'], {
+    stdio: [process.stdin, 'pipe', process.stderr]
```

Comment:
`['inherit', 'pipe', 'inherit']`? Or is there a reason you wrote it like this?

Reply:
Oh, I didn't know you could write `'inherit'` for individual streams. I'll change it.